clerk/tokens.go

```diff
@@ -132,6 +134,16 @@ func verifyTokenParseClaims(parsedToken *jwt.JSONWebToken, key interface{}, sess
 	return parsedToken.Claims(key, sessionClaims, options.customClaims)
 }
 
-func isValidIssuer(issuer string) bool {
+func isValidIssuer(issuer string, options *verifyTokenOptions) bool {
```

Comment:
:upside_down_face: Alternative API, based on the builder pattern

```go
type issuer struct {
  iss string
  isSatellite bool
  proxyURL string
}

func newIssuer(iss string) *issuer {}

func (iss *issuer) ForSatelliteDomain() *issuer {
  iss.isSatellite = true
  return iss
}

func (iss *issuer) WithProxyURL(proxyURL string) *issuer {
  iss.proxyURL = proxyURL
  return iss
}

func (iss *issuer) Valid() bool {
  // use the struct's fields accordingly
}

// Usage
// basic case
// newIssuer("whatever").Valid()
// for satellites
// newIssuer("whatever").ForSatelliteDomain().Valid()
```

Reply:
Thank you!